home/modules/contribute/examples/kv-metrics.json

```diff
@@ -0,0 +1,32 @@
+{
+    "kv_bg_batch_size": {
+        "help": "Batch size for background fetches",
+        "type": "gauge",
+        "unit": "count"
+    },
+    "kv_bg_load_seconds": {
+        "help": "bg fetches waiting for disk",
+        "type": "gauge",
+        "unit": "seconds"
+    },
+    "kv_bg_wait_seconds": {
+        "help": "bg fetches waiting in the dispatcher queue",
+        "type": "gauge",
+        "unit": "seconds"
+    },
+    "kv_cmd_duration_seconds": {
+        "help": "",
```

Comment:
(minor) Missing help for this example

Reply:
That's in the original data, so left as-is to test 👍 
I might remove the field entirely, to check that the templating handles missing fields gracefully...